x/allocation/keeper/abci.go

```diff
@@ -95,7 +96,7 @@ func (k Keeper) EndBlocker(ctx sdk.Context) {
 		// set pending cellar update
 		k.SetPendingCellarUpdate(ctx, types.CellarUpdate{
 			InvalidationNonce: invalidationNonce,
-			Vote:              &wv,
+			Vote:              &winningVotes[i],
```

Comment:
This change seems unnecessary.

Reply:
You would think. This is a quirk of Go loop iterators: https://stackoverflow.com/questions/62446118/implicit-memory-aliasing-in-for-loop

Reply:
Oh man. This also explains the e.g. `v := v` stuff I saw which seemed totally unnecessary.